### PR TITLE
Delete temporary files for tests gpfdist2 and gpfdist2_compress

### PR DESCRIPTION
**Pull request by dkovalev1**
This patch adds cleanup of temporary files used for gpfdist tests.

Tests gpfdist2 and gpfdist2_compress does not clear temporary files used to
check writable external tables. Those files are appended by the next test run 
using same source tree, and results does not match.

It's not a problem in CI, where docker image built with a clean directory, but
can be a problem on a developer's machine, if tests run more than once.

### CODE REVIEW

File: src/bin/gpfdist/regress/input/gpfdist2.source [hunk 1]
```diff
@@ -23,6 +23,15 @@ CREATE TABLE REG_REGION (R_REGIONKEY INT, R_NAME CHAR(25), R_COMMENT VARCHAR(152
...
 -- select * from check_env;
 
+-- Helper to remove the output files
+DROP EXTERNAL WEB TABLE IF EXISTS clean_out_files;
+CREATE EXTERNAL WEB TABLE clean_out_files (x text)
+EXECUTE E'(rm -f @abs_srcdir@/data/gpfdist2/lineitem.tbl.out.zst @abs_srcdir@/data/gpfdist2/test_quote.csv)'
+on SEGMENT 0
+FORMAT 'text';
+
+SELECT * FROM clean_out_files;
```
Comment (RekGRpth) [inline src/bin/gpfdist/regress/input/gpfdist2.source:33]: maybe
```suggestion
SELECT * FROM clean_out_files;
DROP EXTERNAL WEB TABLE clean_out_files;
```

Comment (dkovalev1) [reply src/bin/gpfdist/regress/input/gpfdist2.source:33]: sure

File: src/bin/gpfdist/regress/input/gpfdist2.source [hunk 2]
```diff
@@ -23,6 +23,17 @@ CREATE TABLE REG_REGION (R_REGIONKEY INT, R_NAME CHAR(25), R_COMMENT VARCHAR(152
...
```
Comment (red1452) [inline src/bin/gpfdist/regress/input/gpfdist2.source:33]: Why have you done this before test? What about move it to the end of the test?

Comment (dkovalev1) [reply src/bin/gpfdist/regress/input/gpfdist2.source:33]: I can see 2 pros of cleaning before:
- we are tolerant to a state left from previous runs 
- after the run there are artifacts left, so we can analyze them postmortem
Also, in the other file in this PR there are some artifacts which also cleaned before the test.

**Review by red1452 — APPROVED** [review]
Rename title of PR to imperative

**Review by bimboterminator1 — APPROVED** [review]

**Pull request merged**